packages/hash/api/src/graphql/resolvers/util.ts

```diff
@@ -80,3 +80,23 @@ export const dataTypeQueryDepth = (info: GraphQLResolveInfo): number => {
 export const propertyTypeQueryDepth = (info: GraphQLResolveInfo): number => {
   return queryDepthByFieldName(info, "referencedPropertyTypes");
 };
+
+/**
+ * Returns the requested query depth for referenced link types
+ *
+ * @param info the 'info' fourth argument to all resolvers
+ * @returns the depth requested
+ */
+export const linkTypeQueryDepth = (info: GraphQLResolveInfo): number => {
+  return queryDepthByFieldName(info, "referencedLinkTypes");
+};
+
```

Comment:
I'd prefer sticking with the functions, as the functions are called in quite a few places. Passing the names everywhere is verbose. Additionally, we can change the default values for some functions as needed (e.g. for entities we probably want to set it to `1` instead of `255`).

Reply:
> Additionally, we can change the default values for some functions as needed (e.g. for entities we probably want to set it to 1 instead of 255)

I hadn't considered that, makes sense 👍